projects/cds/src/cms-components/merchandising/carousel/merchandising-carousel/merchandising-carousel.module.ts

```diff
@@ -0,0 +1,30 @@
+import { CommonModule } from '@angular/common';
+import { NgModule } from '@angular/core';
+import { RouterModule } from '@angular/router';
+import { CmsConfig, ConfigModule, UrlModule } from '@spartacus/core';
+import {
+  CarouselModule,
+  MediaModule,
+} from '../../../../../../storefrontlib/src/shared/components/index';
```

Comment:
Please use `@spartacus/storefront` instead

Reply:
Removed this file for this PR, will be added for the cms component PR